pwiz_tools/Skyline/Model/DdaSearch/MsFraggerSearchEngine.cs

```diff
@@ -451,67 +451,67 @@ private void EnsureFastaHasDecoys()
-diagnostic_fragments = 			# [nglycan/labile search_mode only]. Specify diagnostic fragments of labile mods that appear in the low m/z region. Only used if diagnostic_intensity_filter > 0.
-
-clip_nTerm_M = 1			# Specifies the trimming of a protein N-terminal methionine as a variable modification (0 or 1).
-
-use_all_mods_in_first_search = 0
-allow_multiple_variable_mods_on_residue = 0
-max_variable_mods_combinations = 5500			# Maximum number of modified forms allowed for each peptide (up to 65534).
-
-output_format = pepxml_pin			# File format of output files (tsv, pin, pepxml, tsv_pin, tsv_pepxml, pepxml_pin, or tsv_pepxml_pin).
-output_report_topN = 1			# Reports top N PSMs per input spectrum.
-output_max_expect = 50			# Suppresses reporting of PSM if top hit has expectation value greater than this threshold.
-report_alternative_proteins = 1			# Report alternative proteins for peptides that are found in multiple proteins (0 for no, 1 for yes).
-
-precursor_charge = 1 4			# Assumed range of potential precursor charge states. Only relevant when override_charge is set to 1.
-override_charge = 0			# Ignores precursor charge and uses charge state specified in precursor_charge range (0 or 1).
-
-digest_min_length = 5			# Minimum length of peptides to be generated during in-silico digestion.
-digest_max_length = 60			# Maximum length of peptides to be generated during in-silico digestion.
-digest_mass_range = 200.0 5000.0			# Mass range of peptides to be generated during in-silico digestion in Daltons.
-max_fragment_charge = 2			# Maximum charge state for theoretical fragments to match (1-4).
-
-track_zero_topN = 0			# Track top N unmodified peptide results separately from main results internally for boosting features.
-zero_bin_accept_expect = 0			# Ranks a zero-bin hit above all non-zero-bin hit if it has expectation less than this value.
-zero_bin_mult_expect = 1			# Multiplies expect value of PSMs in the zero-bin during  results ordering (set to less than 1 for boosting).
-add_topN_complementary = 0			# Inserts complementary ions corresponding to the top N most intense fragments in each experimental spectra.
-
-minimum_peaks = 15			# Minimum number of peaks in experimental spectrum for matching.
-use_topN_peaks = 50			# Pre-process experimental spectrum to only use top N peaks.
-min_fragments_modelling = 2			# Minimum number of matched peaks in PSM for inclusion in statistical modeling.
-min_matched_fragments = 4			# Minimum number of matched peaks for PSM to be reported.
-minimum_ratio = 0.01			# Filters out all peaks in experimental spectrum less intense than this multiple of the base peak intensity.
-clear_mz_range = 0.0 0.0			# Removes peaks in this m/z range prior to matching.
-
-add_Cterm_peptide = 0.000000
-add_Nterm_peptide = 0.000000
-add_Cterm_protein = 0.000000
+data_type = 0			# Data type (0 for DDA, 1 for DIA, 2 for DIA-narrow-window).

```

Comment:
Are these line ending changes? Consider excluding those from the PR to reduce noise. (they might also be important to MSFragger, I'm not sure).

Reply:
Yeah, those happened without me noticing. I'll change them back, and add a comment
        // N.B. leave these line endings alone, they may matter to MsFragger
 but if that's really the case, this seems like an accident waiting to happen (again). This would be better off using \n or \r or whatever.

Reply:
Actually I think it's OK to leave it as is - it gets run through a StringBuilder so line endings are going to be whatever they're going to be. And if it was a problem for MsFragger wouldn't the tests fail?

Reply:
True, it must be fine if the tests are passing. I like to leave line endings alone because it makes it hard to see the semantic differences, but I just rediscovered GitHub's "Hide whitespace changes" option. They hide it so well...

Reply:
I feel you, but if folks wouldn't commit janky line endings we wouldn't have to worry about non-semantic differences (and we wouldn't be pestered by the idea offering to fix them). Maybe I should make that a new custom code inspection...